src/js/serial_backend.js

```diff
@@ -70,7 +70,8 @@ export function initializeSerialBackend() {
     EventBus.$on('port-handler:auto-select-device', function(device) {
         if (!GUI.connected_to && !GUI.connecting_to
             && ((PortHandler.portPicker.autoConnect && !["manual", "virtual"].includes(device))
-                || Date.now() - rebootTimestamp < REBOOT_CONNECT_MAX_TIME_MS)) {
+                || Date.now() - rebootTimestamp < REBOOT_CONNECT_MAX_TIME_MS)
+                && !GUI.active_tab === 'firmware_flasher') {
```

Comment:
I think remember the old nwjs version reconnected after flashing. What is the case use that covers not reconnecting aflter flashing? Or this is for other reason?

Reply:
Auto-connect was always blocked in firmware flasher due to be able to flash multiple flight controllers.

https://github.com/betaflight/betaflight-configurator/blob/b542569970c4316f3157c7d8d2ff689c8baa1d65/src/js/serial_backend.js#L838